x-pack/plugins/apm/public/components/app/ErrorGroupDetails/index.tsx

```diff
@@ -61,12 +60,9 @@ function getShortGroupId(errorGroupId?: string) {
   return errorGroupId.slice(0, 5);
 }
 
-interface Props {
-  urlParams: IUrlParams;
-  location: Location;
-}
-
-export function ErrorGroupDetailsView({ urlParams, location }: Props) {
+export function ErrorGroupDetails() {
+  const location = useLocation();
```

Comment:
Here and in a few other places `useLocation` is called but the component itself never uses `location` - it is just passed to a child component. It would be a bit better just to call `useLocation` from the child component.
Feel free to clean this up now, or defer it for later.

Reply:
Yeah I went back and forth a bit on this, because ultimately it's still nice to pass props into components so you don't have to rely on hook-based testing (similarly to trying to be careful not to connect everything in Redux). So in places where a component doesn't use location itself _but_ passes it to several other components, it still seems okay to pass it down. But I totally see the argument to just use the hook where the value is needed, too. Will defer this for now.

Reply:
Didn't think about the testing aspect.  Makes sense👍